app/processing.py

```diff
@@ -838,15 +912,15 @@ def document_processing(self):
                         if step.get("stepTaskType") == "assign_task":
                             self.assign_task_helper(step=step, process_id=process_id, process_steps=steps)
                             
-                        update_process(process_id, steps, processing_state)
+                        # update_process(process_id, steps, processing_state)
```

Comment:
Why is this commented, do we need not update the process??

Reply:
I commented it out since the process still gets updated in the next block so I felt it was an unnecessary waste of resources and could definitely affect performance. 

I did not notice any negative effects from commenting it out but it can always be reintroduced if something goes wrong